lib/gui/app/app.js

```diff
@@ -345,6 +359,21 @@ app.controller('HeaderController', function (OSOpenExternalService) {
   this.shouldShowHelp = () => {
     return !process.env.ETCHER_DISABLE_EXTERNAL_LINKS
   }
+
+  /**
+   * @summary Whether to show WiFi dialog link
+   * @function
+   * @public
+   *
+   * @returns {Boolean}
+   *
+   * @example
+   * HeaderController.shouldShowWifi()
+   */
+  this.shouldShowWifi = () => {
+    const parsedEnvVar = _.attempt(JSON.parse, process.env.ETCHER_ENABLE_WIFI_DIALOG)
+    return _.isError(parsedEnvVar) ? false : Boolean(parsedEnvVar)
```

Comment:
There's _many_ other places in the Etcher codebase https://github.com/resin-io/etcher/search?q=Boolean%28process.env&type=Code where a `process.env.variable` is cast either directly or implicitly to a `Boolean`... to ensure that they all get handled consistently, I wonder if it'd be worth adding a shared utility method?

Reply:
Ideally yes, but the env var setting will be partially or wholly replaced – once that's done we'll know whether we need that or not.